Modules/Numerics/Statistics/include/itkScalarImageToCooccurrenceMatrixFilter.h

```diff
@@ -90,7 +90,9 @@ namespace Statistics
  * \ingroup ITKStatistics
  */
 
-template <typename TImageType, typename THistogramFrequencyContainer = DenseFrequencyContainer2>
+template <typename TImageType,
+          typename TMaskType = TImageType,
```

Comment:
I understand that this is a more logical sequence of template parameters. But for painless backwards compatibility it is way better to put the new parameter last. That way other people will not need to change their code.